org.eclipse.jdt.debug.tests/testprograms/FileConditionSnippet2.java

```diff
@@ -0,0 +1,13 @@
+import java.io.File;
+
+public class FileConditionSnippet2 {
+	public static void main(String[] ecs) {
+		int i = 0;
+		File parent = new File("parent");
+		File file = new File(parent,"test");	
+		System.out.println("COMPLETED");
+	}
+
+}
+
+/////////////////////////////////////////
```

Comment:
Please remove this comment line, it is not needed

Reply:
Changes done.